### PR TITLE
Implement register & history

### LINKED ISSUE #29: Implement a function that gets the result of auth and register.

### PR DESCRIPTION
**Pull request by HyeockJinKim**
Closes #29

### CODE REVIEW

File: background/contentScript.js [hunk 1]
```diff
@@ -5,17 +5,24 @@ window.addEventListener(
...
-      chrome.runtime.sendMessage({
-        from: 'content',
-        data: event.data,
-      });
+      let message;
+      if (event.data.message !== undefined) {
+        message = {...event.data, ...event.data.message};
+      } else {
+        message = {...event.data};
+      }
```
Comment (HyeockJinKim) [inline background/contentScript.js:13]: Code for compatibility with the previous version of the 0auth library.

File: background/contentScript.js [hunk 2]
```diff
@@ -5,17 +5,24 @@ window.addEventListener(
...
+        messageQueue.push(message);
+        chrome.storage.local.set({messageQueue});
+      })
     }
   },
   false,
 );
 
-chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+chrome.runtime.onMessage.addListener((request, _sender, _sendResponse) => {
```
Comment (HyeockJinKim) [inline background/contentScript.js:24]: unused parameter.

File: manifest.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "0Auth Extension",
   "version": "1.0",
   "description": "Chrome Extension for 0Auth",
-  "permissions": ["activeTab", "declarativeContent", "storage"],
+  "permissions": ["activeTab", "declarativeContent", "storage", "unlimitedStorage"],
```
Comment (HyeockJinKim) [inline manifest.json:5]: Permission to remove the storage capacity limit stored in the extension.

**Review by HyeockJinKim — COMMENTED** [review]
self review

File: src/components/page/public/index.tsx [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { Container, makeStyles } from '@material-ui/core';
-import { useHistory } from 'react-router-dom';
-import { currentStore } from "../../../stores";
-import { DynamicFormInput } from "@0auth/client";
+import {Container, makeStyles} from '@material-ui/core';
+import {useHistory} from 'react-router-dom';
+import {currentStore, historyStore, propertyStore} from "../../../stores";
+import {DynamicFormInput} from "@0auth/client";
+import {Property, Signature} from "@0auth/message";
```
Comment (anthonyminyungi) [inline src/components/page/public/index.tsx]: I set the ESLint definitely, why didn't it applied here?

File: src/components/page/public/index.tsx [hunk 5]
```diff
@@ -15,40 +16,77 @@ const useStyles = makeStyles(() => ({
...
   }
+  // if (chrome.runtime !== undefined && chrome.runtime.onMessage !== undefined) {
+  //   chrome.runtime.onMessage.addListener((msg, _sender, _response) => {
+  //     switch (msg.data.type) {
+  //       case 'REGISTER':
+  //         register(msg.data.form as DynamicFormInput[]);
+  //         break;
+  //       case 'AUTH':
+  //         auth(msg.data.host as string);
+  //         break;
```
Comment (anthonyminyungi) [inline src/components/page/public/index.tsx]: Could you explain what is this code block of comments?

**Review by anthonyminyungi — CHANGES_REQUESTED** [review]
I reviewed all of the code you uploaded, I think you have to check your formatting environment.

**Review by anthonyminyungi — APPROVED** [review]
LGTM.

**Pull request merged**